tools/utils.py

```diff
@@ -11,13 +11,31 @@
 DOCKER = "docker"
 PODMAN = "podman"
 
+
 def check_output(cmd):
     return subprocess.check_output(cmd, shell=True).decode("utf-8")
 
 
-def get_service_host(service, target=None, domain="", namespace='assisted-installer'):
+def set_profile(target, profile):
+    if target != 'minikube':
```

Comment:
create constant variable

Reply:
why? don't think there is a need to use a constant